From 4b3a470a3676ffcbba2b3d9c2bced524778eb627 Mon Sep 17 00:00:00 2001
From: Wolfgang Bauer <wbauer@tmo.at>
Date: Tue, 10 Apr 2018 16:53:36 +0200
Subject: [PATCH] Revert "Bump libraw minimal version + increase supported file
 extesions"

This reverts commit d4e2ba2b4e42e401171c36d24b4368d58d3a1124.

It raises the minimum libraw version to 0.18.0, but we only have 0.17.1
in Leap 42.3.
---
 CMakeLists.txt |  2 +-
 src/rawfiles.h | 10 +++++-----
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e0dee29..0934b9c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -12,7 +12,7 @@ message(STATUS "Starting CMake configuration for: ${PROJECT_NAME}")
 set(CMAKE_MIN_VERSION   "2.8.12")
 set(ECM_MIN_VERSION     "1.1.0")
 set(REQUIRED_QT_VERSION "5.2.0")
-set(LIBRAW_MIN_VERSION  "0.18")
+set(LIBRAW_MIN_VERSION  "0.16")
 
 cmake_minimum_required(VERSION ${CMAKE_MIN_VERSION})
 
diff --git a/src/rawfiles.h b/src/rawfiles.h
index 1309fbd..24ea979 100644
--- a/src/rawfiles.h
+++ b/src/rawfiles.h
@@ -27,7 +27,6 @@
 #define RAW_FILES_H
 
 // NOTE: extension list Version 1 and 2 are taken from http://www.cybercom.net/~dcoffin/dcraw/rawphoto.c
-// NOTE: extension list Version 5 is taken from https://github.com/MrKepzie/Natron/issues/408#issuecomment-84959815
 
 //                                           Ext    Descriptions From
 //                                                  www.file-extensions.org
@@ -83,17 +82,18 @@ static const char raw_file_extentions[] =
 
 // NOTE: VERSION 4
 
-                                          "*.srw " // Samsung Raw Image Format.
+                                          "*.srw "; // Samsung Raw Image Format.
 
-// NOTE: VERSION 5
+/* TODO: check if these format are supported
                                           "*.drf "  // Kodak Digital Camera Raw Image Format.
                                           "*.dsc "  // Kodak Digital Camera Raw Image Format.
                                           "*.ptx "  // Pentax Digital Camera Raw Image Format.
                                           "*.cap "  // Phase One Digital Camera Raw Image Format.
                                           "*.iiq "  // Phase One Digital Camera Raw Image Format.
-                                          "*.rwz "; // Rawzor Digital Camera Raw Image Format.
+                                          "*.rwz "  // Rawzor Digital Camera Raw Image Format.
+*/
 
 // increment this number whenever you change the above string
-static const int raw_file_extensions_version = 5;
+static const int raw_file_extensions_version = 4;
 
 #endif // RAW_FILES_H
-- 
2.13.6
